CHANGELOG.md

```diff
@@ -12,7 +12,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   - CI/CD files and noxfile syntax
 - `WebsocketManager` Backend
 - New extra install `-E websockets`, additionally a convenience `-E all` option
-- `WebsocketManager` saves response headers on connect
```

Comment:
did you mean to remove this line?

Reply:
Yeah, I removed the response-header-saving functionality in favor of `connect_hook`s on the websocket manager